[OSDOCS#17176]: Make changes as per the errors and warnings for CQA-3 (#114266)
openshift/openshift-docs: modules/security-platform-certificates.adoc

//
// * security/container_security/security-platform.adoc

:_mod-docs-content-type: CONCEPT
[id="security-platform-certificates_{context}"]
= Managing certificates for the platform

[role="_abstract"]

> {product-title} has multiple components within its framework that use REST-based HTTPS communication leveraging encryption via TLS certificates. {product-title}'s installer configures these certificates during installation. There are some primary components that generate this traffic:

Reviewer: ```suggestion
{product-title} has multiple components within its framework that use REST-based HTTPS communication leveraging encryption via TLS certificates. You can configure these certificates during installation. 

There are some primary components that generate this traffic:
```
